### PR TITLE
Docker topic for Docs

### PR DESCRIPTION
**Pull request by agnar-nomad**
In this PR, I am adding a `.md` file describing how to Dockerise a waku app. This short article is supposed to be a part of future documentation.

### CODE REVIEW

File: docs/guides/docker.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+
+## Dockerise a Waku app
+Bundle your Waku app into a Docker container for portability and easy deployment in various environments.
+
+### Prerequisites
+Make sure you have Docker and Node.js installed on your machine. I used `yarn` as my packager manager, customise setup to your manager. We will prepare the image and container in 3 easy steps for reproducible results. Create 3 files in your folder root: `Dockerfile`, `.dockerignore`, `docker-compose.yml`.
```
Comment (dai-shi) [inline docs/guides/docker.md:6]: > I used `yarn` as my packager manager

It's a weird phrase as a guide. Something like "In this guide, we use `yarn`..."

**Review by dai-shi — COMMENTED** [review]
@sandren would you give any advice on writing guides?

File: docs/guides/docker.md (hunk 1 above)
Comment (sandren) [reply docs/guides/docker.md:6]: It should be changed to use `npm` instead for consistency.

**Comment by sandren** [comment]
@dai-shi It looks good. I would just suggest that we standardize the format to MDX with the following intro:

```mdx
---
slug: guide/[slug]
title: Title
description: One sentence description.
---

# Title
```

File: docs/guides/docker.md (hunk 1 above)
Comment (agnar-nomad) [reply docs/guides/docker.md]: The sentence has been removed.
All mentions of `yarn` were replaced by `npm`

**Comment by agnar-nomad** [comment]
I incorporated all all the above notes.
I added the mdx header as requested, however the subsequent article title is put as an H2, for consistency with the other mdx files in the repo.

**Review by dai-shi — COMMENTED** [review]
Thanks for the update. Please run prettier to format docs.

**Comment by agnar-nomad** [comment]
Should be formatted correctly now

**Review by dai-shi — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**